pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/RecommenderConstants.java

```diff
@@ -71,6 +71,7 @@ public static class BloomFilterRule {
 
   public static class RangeIndexRule {
     public static final double DEFAULT_THRESHOLD_MIN_PERCENT_RANGE_INDEX = 0.4;
+    public static final double DEFAULT_THRESHOLD_MIN_CARDINALITY_RANGE_INDEX = 2;
```

Comment:
Do we have any guideline on setting this default threshold to 2? Could we add some description on that?

Reply:
Put two here for some buffer, it is a trade-off between false positive and false negative. We will have to tune this in production (the parameters here are all tunable).